### ISSUE TITLE
Dynamic Lights - Crippling FPS drop

### ISSUE THREAD

**Issue by AlleahSTAR** [AtomicStryker/atomicstrykers-minecraft-mods#23]
There is a marked FPS drop when looking toward spawners with entities with a light value in DynamicLights_entityClasses.cfg. The game becomes unplayable.

This mod is a little intensive for my system/mod pack, but it's perfectly playable and fine (with Optifine).
I set blazes and Thaumcraft's wisp to put out light and, if I'm looking toward their location within approx 50-100 blocks, the game seems to progressively eat more and more memory, until it's maxed out. FML says nothing about a memory leak, or anything at all, for that matter; no errors in my console.

**Details:**
    Minecraft Version: 1.7.10
    Operating System: Linux (amd64) version 3.13.0-24-generic
    Java Version: 1.8.0_60, Oracle Corporation
    Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
    Memory: 1280603112 bytes (1221 MB) / 3077869568 bytes (2935 MB) up to 3113877504 bytes (2969 MB)
    Video Card: Nvidia GeForce GTS 450 
    Driver Version: nvidia-346.82

**Comment by AtomicStryker** [comment]
It is very resource intensive. You can disable modules that stress your PC too much. Unless this is an issue limited to the entityClasses module (which due to the moving nature of entities is the single most stressfull element of the mod) and not lights in general, it is not a bug or rather i cannot make it any more efficient than it already is.

**Comment by AlleahSTAR** [comment]
It is specific to the entityClasses module as it relates to spawners. I have no significant problems with entities emitting light and moving about the world, fighting them, etc., but as soon as one of those entities is in a spawner cage emitting light, there is a severe FPS drop that leads to the game eating all possible memory and locking up.

**Comment by AtomicStryker** [comment]
You are talking about the model inside mob spawner blocks? But those aren't entities and don't shine light through dynamic lights

**Comment by AlleahSTAR** [comment]
I am. I can't say I noticed light being emitted from those models in the cages, either, but for some reason the setting still causes a huge FPS drop. 

In the entityClasses module, if I set blazes to 7 (for example), then load up the game and look at a blaze spawner, my FPS drops steeply-- this is _without_ any blazes spawning. The only way to get rid of the FPS drop issue is to a) break the spawner or b) set blazes back to 0 light in the entityClasses module. Note: spawner cages don't cause me any FPS issues, normally.

Just as a comparison, I can set XP orbs to emit light in the entityClasses module and have a lot of XP on the ground without much FPS loss.

This seems like a very strange issue, especially given that dynamic lights shouldn't be interacting with those models in spawner blocks. Is there any other info I could give you that could help?
